elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -374,13 +399,9 @@ class PfeAccordion extends PFElement {
     }
   }
 
-  _transitionEndHandler(evt) {
-    const header = evt.target.previousElementSibling;
-    if (header) header.classList.remove("animating");
-
-    evt.target.style.height = "";
-    evt.target.classList.remove("animating");
-    evt.target.removeEventListener("transitionend", this._transitionEndHandler);
+  _transitionEndHandler() {
+    clearTimeout(this._animate._tId);
```

Comment:
The event handler now just clears out the timeout to let the rest of the processing continue